src/hsl_ma57.jl

```diff
@@ -437,17 +437,19 @@ by solving the saddle-point system
     [ I  A' ] [ x ]   [ 0 ]
     [ A     ] [ y ] = [ b ].
 """
-function ma57_min_norm(A :: SparseMatrixCSC{T,Ti}, b :: Vector{T}) where {T <: Ma57Data, Ti <: Integer}
+function ma57_min_norm(A :: SparseMatrixCSC{T,Ti}, b :: Array{T}) where {T <: Ma57Data, Ti <: Integer}
   (m, n) = size(A)
   K = [ speye(T, n)  spzeros(T, n, m) ; A  0.0 * speye(T, m) ]
+  M = Ma57(K)
+  ma57_factorize(M)
   rhs = [ zeros(T, n) ; b ]
-  xy57 = ma57_solve(K, rhs)
+  xy57 = ma57_solve(M, rhs)
```

Comment:
Is there an advantage to doing calling `factorize` explicitly?

Reply:
I’m realizing that I should have added min norm and least squares tests to the test. Will do tonight. I added the factorize because I wasn’t able to call ma57_solve on a sparse matrix, I think it needed to be a factorized Ma57 object first